packages/governance/src/multiPluralityVoteCounter.js

```diff
@@ -0,0 +1,242 @@
+    const tally = Array(positions.length).fill(0n);
+
+    for (const { chosen, shares } of allBallots.values()) {
+      const positionIndex = positions.findIndex(p => keyEQ(p, chosen));
+      if (positionIndex < 0) {
+        spoiled += shares;
+      } else {
+        tally[positionIndex] += shares;
+      }
+    }
+
+    /** @type { VoteStatistics } */
+    const stats = {
+      spoiled,
+      votes: allBallots.getSize(),
+      results: tally.map((_, i) => ({
+        position: positions[i],
+        total: tally[i],
+      })),
+    };
+
+    tallyPromise.resolve(stats);
+
+    if (!makeQuorumCounter(threshold).check(stats)) {
+      outcomePromise.reject('No quorum');
+      /** @type {OutcomeRecord} */
+      const voteOutcome = {
+        question: details.questionHandle,
+        outcome: 'fail',
+        reason: 'No quorum',
+      };
+      E(publisher).publish(voteOutcome);
+      return;
+    }
+
+    const sortedPositions = stats.results.sort((a, b) => {
+      if (a.total < b.total) return 1;
+      if (a.total > b.total) return -1;
+      return 0;
+    });
+
+    const winningPositions = [];
+    for (const position of sortedPositions) {
+      if (position.total > 0n) {
+        if (winningPositions.length < questionSpec.maxWinners) {
+          winningPositions.push(position);
+        } else if (
+          winningPositions[winningPositions.length - 1].total === position.total
+        ) {
+          winningPositions.push(position);
+        }
+      }
+    }
+
+    if (winningPositions.length === 0) {
+      outcomePromise.resolve(questionSpec.tieOutcome);
+    } else if (winningPositions.length <= questionSpec.maxWinners) {
+      const outcome = winningPositions.map(p => p.position);
+      outcomePromise.resolve(outcome);
+    } else {
+      const tieIndex = questionSpec.maxWinners - 1;
+      const untiedPositions = winningPositions.slice(0, tieIndex);
+
+      const tiedPositions = winningPositions.slice(tieIndex);
+      const tieWinner = tiedPositions[crypto.randomInt(tiedPositions.length)];
+
+      const outcome = untiedPositions.concat(tieWinner).map(p => p.position);
+      outcomePromise.resolve(outcome);
+    }
+
+    E.when(outcomePromise.promise, position => {
+      /** @type { OutcomeRecord } */
+      const voteOutcome = {
+        question: details.questionHandle,
+        position,
+        outcome: 'win',
+      };
+      return E(publisher).publish(voteOutcome);
+    });
+  };
+
+  const closeFacet = makeHeapFarInstance(
+    'MultiPluralityVoteCounter close',
+    VoteCounterCloseI,
+    {
+      closeVoting() {
+        isOpen = false;
+        countVotes();
+      },
+    },
+  );
+
+  const creatorFacet = makeHeapFarInstance(
+    'MultiPluralityVoteCounter creator',
+    VoteCounterAdminI,
+    {
+      submitVote(voterHandle, chosenPositions, shares = 1n) {
+        assert(chosenPositions.length === 1, 'only 1 position allowed');
+        const [position] = chosenPositions;
```

Comment:
`chosenPositions` might have multiple positions, so you have to check that they're all included in `positions`, and add `chosenPositions` to `completedBallot`.